packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx

```diff
@@ -1,14 +1,14 @@
-import {
+import type {
     ERC1155TokenAssetDetailed,
     ERC721TokenAssetDetailed,
     ChainId,
     CollectibleProvider,
-    resolveCollectibleLink,
 } from '@masknet/web3-shared'
 import { memo } from 'react'
 import { Link, makeStyles, Typography } from '@material-ui/core'
 import { MaskColorVar } from '@masknet/theme'
 import { CollectiblePlaceholder } from '../CollectiblePlaceHolder'
+import { resolveCollectibleLink } from '../../../../../../maskbook/src/plugins/Wallet/pipes'
```

Comment:
It is not allowed to import from the maskbook package into the dashboard package

Reply:
Yes, please move it to some shared package.

Reply:
`resolveCollectibleLink` doesn't seem to rely on any context from `maskbook`, maybe we do the refactoring of moving all plugins into an individual package later.

Reply:
`resolveCollectibleLink` is pipe. IMO this move to a utils package like `web3-shared` would be better.

Reply:
It looks like we separate plugin code into different packages. It's hard to maintain.